docs/specifications/1-tenets.md

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
I would recommend writing everything in a single page (at least initially) to keep it simple.

Reply:
I personally have a preference for smaller files in higher quantities. But not opposed to being pragmatic. The organization of stuff will *definitely* change as the contents get fleshed out... right now I'm mostly trying to manage my own cognitive load / mitigate blank page syndrome / force myself to minimize "things you have to know for this to make sense" (aka - I would like all sections to make sense on their own as much as possible).